vispy/util/fetching.py

```diff
@@ -90,7 +96,10 @@ def get_testing_file(fname, directory=None, force_download=False):
     fname : str
         The path to the file on the local system.
     """
-    _url_root = 'https://github.com/vispy/test-data/raw/master/'
+    global test_data_commit_id
```

Comment:
I just want it to be easy to find since it needs to be updated whenever we change the test-data repo.